conans/test/integration/cmake_magic_lines_test.py

```diff
@@ -0,0 +1,159 @@
+import os
+import subprocess
+import unittest
+
+from conans.test.utils.tools import TestClient
+from nose.plugins.attrib import attr
+
+
+hello_cpp = """
+#include <iostream>
+#include "hello.h"
+
+void hello(){
+    std::cout << "Hello World!" << std::endl;
+}
+"""
+
+hello_h = "void hello();"
+
+conanfile = """
+from conans import ConanFile, CMake
+
+
+class HelloConan(ConanFile):
+    name = "hello"
+    version = "1.0"
+    exports_sources = "hello/*"
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.insert_conan_build_info("hello")
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="hello")
+        self.copy("*hello.lib", dst="lib", keep_path=False)
+        self.copy("*.dll", dst="bin", keep_path=False)
+        self.copy("*.so", dst="lib", keep_path=False)
+        self.copy("*.dylib", dst="lib", keep_path=False)
+        self.copy("*.a", dst="lib", keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = ["hello"]
+"""
+
+cmake = """
+cmake_minimum_required(VERSION 2.8)
+PROJECT(MyHello)
+ADD_LIBRARY(hello hello.cpp)
+"""
+
+
+@attr("slow")
```

Comment:
Not another slow test for a "replace" functionality. Just check the files are there.